### DISCUSSION TITLE
Can this be upgraded to support intl: ^0.19.0?

### DISCUSSION THREAD

**Discussion by monggos** [flutter-form-builder-ecosystem/form_builder_validators#75] [General]
I am having some issues on some updating my flutter app because this package supports up to "intl: ^0.18.1". Your package such a huge help with us. THANK YOU!

**Comment by deandreamatias** [comment]
Take a look to this issue https://github.com/flutter-form-builder-ecosystem/form_builder_validators/issues/73#issuecomment-1854525111
